src/main/java/ch/wisv/chue/HueService.java

```diff
@@ -13,15 +16,18 @@
 
 import javax.annotation.PostConstruct;
 import javafx.scene.paint.Color;
-import java.util.*;
 import java.util.List;
 import java.util.stream.Collectors;
 
 @Service
 public class HueService {
-    private static final int MAX_HUE = 65535;
+    public static final int MAX_HUE = 65535;
```

Comment:
Deze is nu dus twee keer gedefinieerd? (Ook in `HueCommand`)

Is dat intentionally, of zouden die twee van elkaar af kunnen hangen?